<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Rtc</class>
 <widget class="QWidget" name="Rtc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/image/main_menu/background2.png);</string>
  </property>
  <widget class="QPushButton" name="close_Btn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>116</width>
     <height>58</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>116</width>
     <height>58</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>116</width>
     <height>58</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;outline:none;border:0px;
background-image: url(:/image/FM/close_btn.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>20</y>
     <width>32</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
background-image: url(:/image/main_menu/WIFI.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>217</x>
     <y>80</y>
     <width>849</width>
     <height>58</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);
background-image: url(:/image/FM/biaoti.png);</string>
   </property>
   <property name="text">
    <string>时间设置</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>20</y>
     <width>15</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
background-image: url(:/image/main_menu/bt_down.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="time_dis_lab">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>20</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
background-image: url(:/image/main_menu/4g.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="yEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>340</y>
     <width>100</width>
     <height>46</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);
border:1px solid #4091BF;border-radius:10px;</string>
   </property>
   <property name="text">
    <string>2023</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>301</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>分</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>301</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>秒</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>301</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>时</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="minEdit">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>340</y>
     <width>100</width>
     <height>46</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);
border:1px solid #4091BF;border-radius:10px;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="mEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>340</y>
     <width>100</width>
     <height>46</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);
border:1px solid #4091BF;border-radius:10px;</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="dEdit">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>340</y>
     <width>100</width>
     <height>46</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);
border:1px solid #4091BF;border-radius:10px;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>301</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>年</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>301</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>日</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="hEdit">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>340</y>
     <width>100</width>
     <height>46</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);
border:1px solid #4091BF;border-radius:10px;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="SecEdit">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>340</y>
     <width>100</width>
     <height>46</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);
border:1px solid #4091BF;border-radius:10px;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>301</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>月</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_setTimer">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>340</y>
     <width>161</width>
     <height>46</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;outline:none;border:0px;
background-image: url(:/image/BT/btn_img.png);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>设置时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>460</y>
     <width>161</width>
     <height>46</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);
background-image: url(:/image/BT/btn_img.png);</string>
   </property>
   <property name="text">
    <string>当前时间</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>421</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>秒</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>421</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>时</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>421</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>分</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>421</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>日</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>421</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>年</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>421</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>月</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="y_label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>460</y>
     <width>100</width>
     <height>46</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);
border:1px solid #4091BF;border-radius:10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="m_label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>460</y>
     <width>100</width>
     <height>46</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);
border:1px solid #4091BF;border-radius:10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="d_label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>460</y>
     <width>100</width>
     <height>46</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);
border:1px solid #4091BF;border-radius:10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="h_label">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>460</y>
     <width>100</width>
     <height>46</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);
border:1px solid #4091BF;border-radius:10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="min_label">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>460</y>
     <width>100</width>
     <height>46</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);
border:1px solid #4091BF;border-radius:10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="sec_label">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>460</y>
     <width>100</width>
     <height>46</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);
border:1px solid #4091BF;border-radius:10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
